helpers.go

```diff
@@ -368,18 +375,23 @@ func InstallPostSymlinks(post *PostInstall, rootfsPath string) llb.RunOption {
 	})
 }
 
-func (s *Spec) GetSigner(targetKey string) *PackageSigner {
+func (s *Spec) GetSigner(targetKey string) (*PackageSigner, Warning) {
```

Comment:
Instead this could return a bool to help simplify understanding what's going on here.
Also would need a doc string to explain what the 2nd return is for.
Since `Warning` isn't really an error this feels a bit strange.

Reply:
We changed this to remove the bool which is why I went with this. But I'll put the bool back

Reply:
The old case was returning `true` if `PackageSIgner` was not nil.
I think what we want is to return a bool when the package signer in the target is superseding the base?